src/composite.c

```diff
@@ -0,0 +1,119 @@
+#include <stdio.h>
+#include <vips/vips.h>
+
+/* Composite images `src` and `dst`.
+ */
+int Composite(VipsObject *context, VipsImage *src, VipsImage *dst,
```

Comment:
PascalCase for classes and camelCase for variable names seems to be near-standard across C++, Java and JavaScript. The Google C++ guidelines use PascalCase for (non-accessor) method names, but I suspect that may have been influenced by the prevalence of C++ for Windows development in the 90s. The Sun Java guidelines use camelCase for method names, which JavaScript seems to have inherited. 

The only conclusion I've been able to reach is that consistency within a code base is more important than consistency across code bases.